benchmark/sirun/Dockerfile

```diff
@@ -37,3 +37,14 @@ RUN mkdir -p /usr/local/nvm \
 	&& nvm install --no-progress 20.4.0 \
 	&& nvm alias default 18 \
 	&& nvm use 18
+
+RUN mkdir /opt/insecure-bank-js
+RUN git clone --depth 1 https://github.com/hdiv/insecure-bank-js.git /opt/insecure-bank-js
+
+WORKDIR /opt/insecure-bank-js
+RUN sed -i "s#global.sequelize = new Sequelize('sqlite::memory:', {#global.sequelize = new Sequelize('sqlite::memory:', {\nlogging:false,\n#g" app.js
```

Comment:
To avoid the sed command, we could extract the `logging` property to an environment variable